tests/parquet_test.py

```diff
@@ -16,20 +17,26 @@ def test_parquet(self):
         # get the dset from the dictionary in multi-locale cases
         for f in glob.glob('pq_test*'):
             os.remove(f)
-            self.assertTrue((ak_arr ==  pq_arr).all())
+            a = ak_arr.to_ndarray().sort()
+            b = pq_arr.to_ndarray().sort()
+            self.assertTrue(a ==  b)
 
     def test_multi_file(self):
         adjusted_size = int(SIZE/NUMFILES)*NUMFILES
         test_arrs = []
+        elems = ak.randint(0, 2**32, adjusted_size)
+        per_arr = int(adjusted_size/NUMFILES)
         for i in range(NUMFILES):
-            test_arrs.append(ak.randint(0, 2**32, int(adjusted_size/NUMFILES)))
+            test_arrs.append(elems[(i*per_arr):(i*per_arr)+per_arr])
             test_arrs[i].save_parquet("pq_test" + str(i), "test-dset")
 
+        a = elems.to_ndarray()
+        a.sort()
         pq_arr = ak.read_parquet("pq_test*", "test-dset")
```

Comment:
Same thing here: we should have the Chapel code sort the filenames from the glob prior to reading. Then you can just do `self.assertTrue((elems == pq_arr).all())` without any `.to_ndarray()` or `.sort()`.